chore: Updating Lookout documentation (#15020)
newrelic/docs-website: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx

      Combining the information from these sections, not only can you get a sense of what is going on, but also where to focus actions to resolve the problem.
  </Collapser>
</CollapserGroup>

## Diagnose anomalies with Lookout [#explore-the-data]

To analyze an application or facet, click a circle, table row, or abnormal golden signal. When clicked, the button opens a diagnostic tool you can use to explore details about your golden signals. The details panel shows a link to the affected [entity](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic), the golden signals over time and deviations of the performance indicator, and recent alert and deployment activity for that entity, when available.

There are up to five tabs available. The details of the individual tabs are outlined further.

### Workload change tab [#workload-change]

> This tab helps you spot patterns in event attributes that might be related to the deviation in the main entity.
> 
> You can select an event type see how the values of its event attributes changed for your entity in the selected time period against the preceding same duration. A timeline visualization is provided to show the sample time period and the preceding comparison time period.

Reviewer: ```suggestion
This tab helps you spot patterns in event attributes that might be related to the deviation in the main entity.

You can select an event type to see how the values of its event attributes changed for your entity in the selected time period against the preceding same duration. A timeline visualization is provided to show the sample time period and the preceding comparison time period.
```

<img
  title="Workload change"
  alt="An image showing the workload change tab"
  src={diagnoseWorkloadChange}
/>

> To visualize and analyze a specific event attribute, select show detailed view.

Reviewer: ```suggestion
To visualize and analyze a specific event attribute, toggle **Show detailed view**.
```

> ### Related changes tab [#related-changed]

Reviewer: ```suggestion
### Related changes tab [#related-changes]
```

> New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high confidence score or a low confidence. This tab displays a table format and, to dig in further, you can click through to the correlated entity. 

Reviewer: ```suggestion
New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity. 
```

> New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity. 

Reviewer: ```suggestion
The Related changes tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity.
```

<img
  title="Related changes"
  alt="An image showing the related changes tab"

>   src={diagnoseRelatedChanges}

Reviewer 2: ```suggestion
  src={diagnoseRelatedChange}
```

Reviewer 2: This should get the build working!

/>

### Seasonality tab [#seasonality]

> The seasonality tab shows the changes in the selected golden signal over time and also surfaces any patterns in the anomalies that we find. Once you select the golden signal you can then view the data over an hour, a day or a week. The anomaly patterns are also presented in a table below the chart and you can click on a link to see more details of the anomaly.

Reviewer: ```suggestion
The Seasonality tab shows the changes in the selected golden signal over time and also surfaces any patterns in the anomalies that we find. Once you select the golden signal you can then view the data over an hour, a day or a week. The anomaly patterns are also presented in a table below the chart and you can click on a link to see more details of the anomaly.
```

<img
  title="Seasonality"
  alt="An image showing the seasonality tab"
  src={diagnoseSeasonality}
/>

### Performance tab [#performance]

> The Performance tab shows charts for other key performance indicators for the selected application or facet. 
> 
> When the target is a New Relic application, we show the top transactions, error classes, external services, and database operations, alongside links to the relevant features. If there is no data for any of these, the charts are not shown.

Reviewer: ```suggestion
The Performance tab shows charts for other key performance indicators for the selected application or facet. When the target is a New Relic application, we show the top transactions, error classes, external services, and database operations, alongside links to the relevant features. If there is no data for any of these, the charts are not shown.
```

### Traces tab [#traces]

> If the entity has distributed traces configured and available in the selected evaluated time window, the **Traces** section is enabled. Each card shows a summary of a trace and can be clicked for details. **Explore all traces** links to the [distributed traces application](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui), filtered to the selected entity.

Reviewer: ```suggestion
The Traces tab is displayed only if the entity has distributed traces configured and available in the selected evaluated time window. Each card shows a summary of a trace and can be clicked for details. **Explore all traces** links to the [distributed traces application](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui), filtered to the selected entity.
```
